Frontend/ui-customer/src/Helpers/Storage/Flow/BaseApp.purs

```diff
@@ -64,3 +69,10 @@ baseAppStorage = do
         setValueToLocalStore SESSION_ID $ generateSessionId unit
     when (countryCode `elem` ["__failed", "(null)"]) do
         setValueToLocalStore COUNTRY_CODE "+91"
+    when (termsVersion `elem` ["__failed", "(null)"]) do
+        setValueToLocalStore T_AND_C_VERSION "1"
+    let isTermsUpdated = fromMaybe 0 (fromString $ getValueToLocalStore T_AND_C_VERSION) < fromMaybe 0 (fromString $ appConfig.termsVersion)
```

Comment:
For the first time toast will not be show, Can you change this condition or update config value

Reply:
For which merchant should I update the config? @kavya-shree-s